src/database/removeHelper.js

```diff
@@ -0,0 +1,121 @@
+"use strict";
+
+var api = require("../api");
+var FirebaseError = require("../error");
+var request = require("request");
+var responseToError = require("../responseToError");
+var utils = require("../utils");
+const logger = require("../logger");
+
+class DatabaseRemoveHelper {
```

Comment:
Are there tests for this class?

Reply:
No.

It is doing relatively straight forward work of calling an endpoint and return the data.
If there is anything wrong here, my manual testing will certainly fail.

Reply:
Done